smoke-tests-otel-starter/spring-boot-common/src/main/java/io/opentelemetry/spring/smoketest/AbstractOtelSpringStarterSmokeTest.java

```diff
@@ -214,12 +214,12 @@ void shouldSendTelemetry() {
 
     // JMX based metrics - test one per JMX bean
     List<String> jmxMetrics =
-        new ArrayList<>(
-            Arrays.asList(
-                "jvm.thread.count",
-                "jvm.memory.used",
-                "jvm.system.cpu.load_1m",
-                "jvm.memory.init"));
+        new ArrayList<>(Arrays.asList("jvm.thread.count", "jvm.memory.used", "jvm.memory.init"));
+
+    double javaVersion = Double.parseDouble(System.getProperty("java.specification.version"));
+    if (javaVersion < 23) {
+      jmxMetrics.add("jvm.system.cpu.load_1m");
+    }
```

Comment:
can you add a comment here why this limitation (even if it's just a TODO to investigate further)?